react/week2/src/components/Timer.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+
+
+class Timer extends Component {
```

Comment:
Can you try to rewrite this file with functions? As you did in other components...